README.md

```diff
@@ -62,13 +62,21 @@ then enforces a specific workflow for managing release branches, changelogs, art
 
 ## Installation
 
-The tool is distributed as an NPM package and can be installed via `npm` or `yarn`:
+### Binary
 
-```bash
-yarn global add @sentry/craft
+`craft` is [distributed as a minified single JS binary](https://github.com/getsentry/craft/releases/latest).
 
-# Or (not preferred):
+### npm (not recommended)
 
+Recommendation is to used this file directly but one can also install `craft` as
+an [NPM package](https://yarn.pm/@sentry/craft) and can be installed via `yarn`
+or `npm`:
+
+```shell
+yarn global add @sentry/craft
```

Comment:
Yarn represent. 😁

Reply:
LOL, it was already preferred before I touched this part :D